src/Illuminate/Support/helpers.php

```diff
@@ -863,3 +863,17 @@ function with($object)
         return $object;
     }
 }
+
+if (! function_exists('loop')) {
```

Comment:
This can't go in here because it depends on components outside the support component.